### DISCUSSION TITLE
How to know the version of kaboom I'm using in my projects?

### DISCUSSION THREAD

**Discussion by Stutz** [replit/kaboom#627] [Ideas]
Hi,
sometimes I get confused about which version of Kaboom I'm using in my projects. There's no way to know that.
To solve this problem, I suggest implementing the version property in the KaboomCtx object, containing the engine version number.

Best regards, Stu

**Comment by lajbel** [comment]
I think you can check in the import or in the package.json

**Reply by Stutz** [reply]
This is true if I'm using package.json. However, this information is indirect. Simply by looking at the code, you can't tell which version of Kaboom it is. The version information in the engine itself is extremely useful in version testing, for example. When I suggested the proposal, I thought of something similar to what happens in the Phaser engine. When the game context is started, information about the version used is printed in the console.log.

![image](https://user-images.githubusercontent.com/354282/188493405-bf467d5a-0ba3-4c7c-9901-995356aa988f.png)

**Reply by lajbel** [reply]
Oh I see, maybe we can do it, thanks

**Comment by lajbel** [comment]
Just added `VERSION` constant on `@next` version :D
